code/controllers/subsystem/ships.dm

```diff
@@ -153,7 +155,8 @@ SUBSYSTEM_DEF(ship)
 			spawn(50)
 
 				broadcast_message("<span class=warning>Enemy ship ([S.name]) fired their [W.name] and hit! Hit location: [target.loc].</span>",error_sound,S) //so the message doesn't get there early
-
+				if(W.attack_data.shield_bust)
+					broadcast_message("<span class=warning>Enemy ships [W.name] pierces the shield.",error_sound,S)
```

Comment:
Rather than broadcasting 2 messages and beeping twice, they should be in one message.
Also the phrasing is weird and it's the possessive form of ship, not plural.